src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java

```diff
@@ -113,31 +115,31 @@ private Map<String, String> getUploadedDocuments(CaseData caseData) {
         Map<String, String> uploadedDocuments = new HashMap<>();
 
         int count = 0;
-        if (caseIssueDecision.getDecisionNotice().equals(NoticeOption.UPLOAD_FROM_COMPUTER)
-            && caseIssueDecision.getDecisionDocument() != null) {
+        if (caseIssueDecision.getDecisionNotice().equals(NoticeOption.UPLOAD_FROM_COMPUTER)) {
 
```

Comment:
It's an enum identity, so can use == instead of .equals

Reply:
Updated condition